### PR TITLE
Feat/203: Telemetry consent

### LINKED ISSUE #203: Support opting out sentry error reporting
We think we shouldn't report to sentry without explicit consent from the users.

### PR DESCRIPTION
**Pull request by notbot00**
Closes #203 

Screenshot 2022-08-15 at 15 54 05

### CODE REVIEW

**Review by dexturr — CHANGES_REQUESTED** [review]
Bunch of linting issues 😢

File: frontend/src/components/transaction-manager/transaction-manager.tsx [hunk 1]
```diff
@@ -17,7 +17,19 @@ const logger = createLogger('TransactionManager')
  * Stores an array of parsed transactions which get passed to a modal
  */
 export function TransactionManager() {
-  const [transactions, setTransactions] = useState<ParsedTx[]>([])
+  const [transactions, setTransactions] = useState<ParsedTx[]>([
+    {
```
Comment (mattrussell36) [inline frontend/src/components/transaction-manager/transaction-manager.tsx:21]: question (blocking): Whys this here?

**Review by mattrussell36 — COMMENTED** [review]
This looks all fine apart from my one comment and some linting issues

File: frontend/src/components/transaction-manager/transaction-manager.tsx (hunk 1 above)
Comment (notbot00) [reply frontend/src/components/transaction-manager/transaction-manager.tsx]: oh no.. i think i accidentally unstashed some changes here from another branch i had. thanks for the catch, removed it now

**Pull request merged**